.github/workflows/release.yml

```
@@ -0,0 +1,158 @@
+
+name: Build release
+
+permissions:
+  actions: read
+  checks: read
+  contents: write
+  deployments: none
+  issues: read
+  packages: write
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: read
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build'
```

```suggestion
        description: 'Or specify a tag to build from'
```

Maybe the action needs to check and update `core/Version.php` otherwise it would be wrong when releasing from a tag?